setup.py

```diff
@@ -54,7 +55,6 @@
     "reana-db>=0.8.0a1,<0.9.0",
     "htcondor==8.9.7",
     "retrying>=1.3.3",
-    "paramiko[gssapi]>=2.6.0",
```

Comment:
Since `paramiko` became optional dependency, perhaps move import later into the function where it is really needed?

```
$ rg paramiko
reana_job_controller/utils.py
16:import paramiko
```

Chances are that nobody would `pip install r-j-controller` outside of REANA, so this should not matter too much... provided we always use `[all]`... but it could be good style.

(E.g. try a new virtual environment, install just like this `pip install .` and check `flake8`...)